lib/glueby/active_record/system_information.rb

```diff
@@ -12,21 +12,74 @@ def self.use_only_finalized_utxo?
         find_by(info_key: "use_only_finalized_utxo")&.int_value != 0
       end
 
+      # Set use_only_finalized_utxo
+      def self.set_use_only_finalized_utxo(status)
+        if exists?(info_key: "use_only_finalized_utxo")
+          current = find_by(info_key: "use_only_finalized_utxo")
+          current.update!(info_value: status)
+        else
```

Comment:
Using `find_by` first can eliminate a query by `exists?` like below. It is more efficient. 

```suggestion
        current = find_by(info_key: "use_only_finalized_utxo")
        if current
          current.update!(info_value: status)
        else
```
Can you please modify `self.set_utxo_provider_default_value(value)` , `self.set_utxo_provider_pool_size(size)`  and `self.set_broadcast_on_background(status)` in the same way.